lib-test/src/main/kotlin/app/cash/nostrino/model/ArbEventContent.kt

```diff
@@ -21,57 +26,75 @@ import io.kotest.property.arbitrary.triple
 
 object ArbEventContent {
 
-  val arbEncryptedDm: Arb<EncryptedDm> = Arb.triple(arbSecKey, arbSecKey, Arb.string()).map { (from, to, message) ->
-    EncryptedDm(from, to.pubKey, message)
+  val arbEncryptedDm: Arb<EncryptedDm> by lazy {
```

Comment:
Made all arbs `by lazy` because order of execution can screw up test initialisation sometimes.